src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/util/AuthorizationEnabledIntegrationTest.java

```diff
@@ -122,6 +122,9 @@
     @Value("${exui.role.pui-case-manager}")
     protected String puiCaseManager;
 
+    @Value("${exui.role.pui-caa}")
+    protected String puiCaa;
+
```

Comment:
why are the roles defined as exui.role? They should just be defined as  -
prd:
    security:
         roles:

Reply:
in the next jira i will do it like you mentioned